unit_tests/test_jc_api_v1.py

```diff
@@ -138,6 +138,28 @@ def test_get_user_no_id(self, mock_systemusers_list):
         with pytest.raises(SystemUserNotFoundError):
             api1.get_user('angela')
 
+    @patch.object(jcapiv1.SearchApi, 'search_systemusers_post')
+    def test_search_users(self, mock_search_systemusers_post):
```

Comment:
could we create a unit test for the AND condition (first & last name)?

Reply:
Good idea. I think I'll add a step to the integration tests that does the same as well

Reply:
Sorry, I'm recovering from a severe case of vacation brain... we already have one such integration test. I'll just make the unit test.

Reply:
Generally, an effective unit test is preferable over an integration test.  In this case i think a unit test is enough to test this functionality.